signs-work-order-map/components/Map.tsx

```diff
@@ -82,19 +105,24 @@ export default function Map({ location, signs }: MapProps) {
       {signPins}
 
       {
-        // showing the location / geolocation will happen in a subsequent issue
-        console.log("location from payload: ", location)
-        // when do we show location vs signs? when there are no signs?
-        /* {location?.latitude && location?.longitude && (
-        <Marker latitude={location.latitude} longitude={location.longitude} />
-      )} */
+        // the add location marker, work will be completed in a following PR
+        // location?.latitude && location?.longitude && (
+        //   <Marker latitude={location.latitude} longitude={location.longitude} />
+        // )
+        console.log(location)
       }
 
       {popupInfo && (
         <SignPopup popupInfo={popupInfo} setPopupInfo={setPopupInfo} />
       )}
 
       <GeocoderControl position="top-left" marker={true} />
+      <GeolocateControl
+        position="top-left"
+        showUserLocation={false}
+        fitBoundsOptions={{ maxZoom: 16, duration: 0 }}
+      />
```

Comment:
When I click this button I get a console error. Not sure if it has anything to do with my location being out of the map bounds 🤔 

<img width="531" alt="Screenshot 2025-06-26 at 2 32 06 PM" src="https://github.com/user-attachments/assets/cb1072ea-191b-442f-a88b-8bc1f3e6d481" />

Reply:
Does it geolocate correctly without the button?

Reply:
Without the button and using my real location, the map pans to the edge of the max bounds for the map. I was able to use Chrome dev tools to override my location with a location in Austin, and it worked as expected 👍 

I did run into this error with my real location and the location override 🤷